src/Weasyprint.Wrapped/Printer.cs

```diff
@@ -51,7 +51,7 @@ public async Task Initialize()
         }
     }
 
-    public async Task<PrintResult> Print(string html)
+    public async Task<PrintResult> Print(string html, CancellationToken cancellationToken = default)
```

Comment:
You need to create another overload. This is a breaking change if you swap out the library without recompiling its dependents.

Reply:
Thank you so much for your contribution @ivan-burlakov and for your helpful feedback @winnerlein.
I've created a new issue #38 for this and will have a look at the code this evening!